### ISSUE TITLE
Multiple mbtiles in dir (docker)

### ISSUE THREAD

**Issue by lyashkoandrei** [maptiler/tileserver-gl#1221]
I'm using Docker for tileserver. Directory with .mbtiles files was created and mapped. But after container started I saw this message

[INFO] Automatically creating config file for ga_bibb.mbtiles
2024-03-26 19:33:26 [INFO] Only a basic preview style will be used.
2024-03-26 19:33:26 [INFO] See documentation to learn how to create config.json file.
2024-03-26 19:33:26 WARN: MBTiles not in "openmaptiles" format. Serving raw data only...

and there is only one tileset on map example.

**Comment by acalcutt** [comment]
Did you look at the thread right below this one? I explained how to do this there
https://github.com/maptiler/tileserver-gl/issues/1209#issuecomment-1994935419

**Comment by lyashkoandrei** [comment]
> Did you look at the thread right below this one? I explained how to do this there [#1209 (comment)](https://github.com/maptiler/tileserver-gl/issues/1209#issuecomment-1994935419)

yes, I read the recommendations. Everything works with the test files. But when I try to connect my tiles, it's just an empty canvas. I changed the borders for the style according to my tiles. Maybe the reason is that these are tiles created in tippecanoe for mapbox? Only after on btn above zoom btns I see my polygons, but the map is still empty.
![image](https://github.com/maptiler/tileserver-gl/assets/40296802/c6ffcd72-89fe-4b4a-a5f6-cda0708d54bd)
